sdk/monitor/monitor-opentelemetry/src/metrics/performanceCounters.ts

```diff
@@ -130,7 +130,7 @@ export class PerformanceCounterMetrics {
       },
     );
     this.processTimeGauge = this.meter.createObservableGauge(
-      PerformanceCounterMetricNames.PROCESS_TIME,
+      PerformanceCounterMetricNames.PROCESS_TIME_STANDARD,
```

Comment:
Are you planning to use PROCESS_TIME_NORMALIZED and EXCEPTION_RATE in a different PR?

Reply:
These two perf counters are already implemented in this file and should be collected.